deployment/ipam.yaml

```diff
@@ -17,6 +17,7 @@ spec:
     metadata:
       labels:
         app: ipam
+        "spiffe.io/spiffe-id": "true"
```

Comment:
the spiffe labels are implemented in https://github.com/Nordix/Meridio-Operator/pull/55, Lionel is waiting for the PR in Meridio to be merged first. Maybe you can rebase on that after that is merged?

Reply:
I don't see a point for doing the spiffe label thing separately. The PR was implemented and tested with labels included, so wouldn't prefer changing that just for the sake of a small PR.